release-please-config.json

```diff
@@ -36,7 +36,6 @@
     },
     "python": {
       "release-type": "python",
-      "draft": true,
```

Comment:
actually this is what was preventing the python automatic publishing (I was manually editing the release to "publish" it). But again, ready for primetime